flexget/plugins/input/npo_watchlist.py

```diff
@@ -201,33 +201,36 @@ def _parse_episodes_npo3(self, task, config, series_name, page, series_info):
 
         entries = list()
         for listItem in page.findAll('div', class_='item'):
```

Comment:
I know this wasn't introduced in this PR but please change `listItem` to `list_item`